### ISSUE TITLE
Update the Data Quality chapter

### ISSUE THREAD

**Issue by schuemie** [OHDSI/TheBookOfOhdsi#115]
The Data Quality Dashboard has matured, and we’re about to launch the CohortDiagnostics package, which I’d like to describe here.

@clairblacketer , what do you think?

**Comment by clairblacketer** [comment]
@schuemie definitely! I agree we have learned a lot since the original edition and the Data Quality chapter could use an update.
